app/frontend/tests/unit/services/formService.spec.js

```diff
@@ -0,0 +1,260 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+import formService from '@/services/formService';
+import { ApiRoutes } from '@/utils/constants';
+
+const mockInstance = axios.create();
+const mockAxios = new MockAdapter(mockInstance);
+
+const zeroUuid = '00000000-0000-0000-0000-000000000000';
+const oneUuid = '11111111-1111-1111-1111-111111111111';
+
+jest.mock('@/services/interceptors', () => {
+  return {
+    appAxios: () => mockInstance
+  };
+});
+
+describe('Form Service', () => {
+  beforeEach(() => {
+    mockAxios.reset();
+  });
+
+  //
+  // Forms
+  //
+  describe('Forms/{id}', () => {
```

Comment:
For service unit tests, it's a better convention to have the description contain the HTTP Method specified:
```js
  describe('GET /forms/{id}', () => {
```

Reply:
Have some other unit tests locally to go in next ticket so will adjust with those